code/datums/supply_packs.dm

```diff
@@ -702,7 +703,8 @@
 					/obj/item/reagent_containers/glass/bottle/pfd = 2,
 					/obj/item/reagent_containers/glass/bottle/pentetic,
 					/obj/item/reagent_containers/glass/bottle/haloperidol,
-					/obj/item/reagent_containers/glass/bottle/ether)
+					/obj/item/reagent_containers/glass/bottle/ether,
+					/obj/item/reagent_containers/emergency_injector/pentetic_acid = 2)
```

Comment:
Not a huge fan of this, as there's a bottle of pent in here anyways, and the rest of the crate is all bottles

Reply:
Done! Moved the pent injector to D.O.C. and cleared it out of the crate!